src/__mocks__/replies.ts

```diff
@@ -0,0 +1,38 @@
+import { user } from './annotations';
+
+const replies = [
```

Comment:
We're mocking this at a very high level in `src`. Is this going to be used everywhere across the codebase? Should it be a fixture or a mock of some other module?

Reply:
It is on the same level as annotations mock and I believe this mock will be handy in `src/features/annotation-thread/AnnotationThread.js` component, so I would leave it as it is.